ci/courtesy-push/courtesy-push.js

```diff
@@ -0,0 +1,78 @@
+const path = require('path');
```

Comment:
Rather than have all the console.log lines commented out, can we have an command line option to turn them on? Or maybe piggy-back on system.debug=true to enable it?

Reply:
Honestly, I don't know why I left them commented out. I'd probably lean towards just uncommenting, there's nothing there that needs to be hidden or should be too big

Reply:
Updated